lib/repeatable.js

```diff
@@ -27,7 +27,7 @@ module.exports = function(Queue) {
     var now = Date.now();
     now = prevMillis < now ? now : prevMillis;
 
-    var nextMillis = getNextMillis(now, repeat.cron, repeat);
+    var nextMillis = getNextMillis(now, repeat);
```

Comment:
maybe an exception should be thrown if the user uses both the milliseconds and the cron expression at the same time.